src/digitRecognition/ErrorFunction.java

```diff
@@ -0,0 +1,232 @@
+                    System.out.println("Errors rate for class " + i + " = " + dec.format(errors)  + "%");
+                    break;
+                }
+                case 6: {
+                    errors = errorFunction(imagesByClass.six,i);
+                    System.out.println("Errors rate for class " + i + " = " + dec.format(errors)  + "%");
+                    break;
+                }
+                case 7: {
+                    errors = errorFunction(imagesByClass.seven,i);
+                    System.out.println("Errors rate for class " + i + " = " + dec.format(errors)  + "%");
+                    break;
+                }
+                case 8: {
+                    errors = errorFunction(imagesByClass.eight,i);
+                    System.out.println("Errors rate for class " + i + " = " + dec.format(errors)  + "%");
+                    break;
+                }
+                case 9: {
+                    errors = errorFunction(imagesByClass.nine,i);
+                    System.out.println("Errors rate for class " + i + " = " + dec.format(errors)  + "%");
+                    break;
+                }
+
+                default: {
+                    return;
+                }
+            }
+        }
+    }
+
+
+
+
+    /********************************************************************************
+     *
+     * errorFunction Method checks the collection and compare label of picture and
+     * it accordance to the collection. For example if in collection with drawings of
+     * number five would found the image labled as 3- it'll means error.
+     *
+     * @param col collection which should be checked
+     * @param realValue is value of label which should be at collection
+     * @return  errorsRate in %
+     *******************************************************************************/
+    double errorFunction(ArrayList<ImageIDX> col, int realValue) {
+        double errors = 0;
+        double errorsRate =0;
+
+        if ( col.isEmpty() ) {
+            System.out.println("Collection is empty");
+            return 0;
+        }
+
+        for (ImageIDX aCol : col) {
+            if (aCol.label != realValue) {
+                errors++;
+            }
+        }
+
+        errorsRate = ( errors/(double)col.size() )*100;
+
+        return errorsRate;
+    }
+
+
+
+    /********************************************************************************
+     *
+     * randomClassifier Method  assigns random image from test collection
+     * to class from zero  to nine
+     *
+     * @return  result - true if ok, false if something goes wrong
+     *******************************************************************************/
+    boolean randomClassifier() {
+        for (ImageIDX aTestCol : testCol) {
+            Random random = new Random();
+            int rand = random.nextInt(10);
+            switch (rand) {
+                case 0: {
+                    imagesByClass.zero.add(aTestCol);
```

Comment:
If you should recognize 256 numbers rather than 9 you will complicate this switch? It is better to write more generic code without large potentially growing switches.